modules/container-configbaker/scripts/bootstrap/demo/init.sh

```diff
@@ -19,6 +19,13 @@ echo ""
 echo "Setting DOI provider to \"FAKE\"..."
 curl -sS -X PUT -d FAKE "${DATAVERSE_URL}/api/admin/settings/:DoiProvider"
 
+API_TOKEN=$(grep apiToken "/tmp/setup-all.sh.out" | jq ".data.apiToken" | tr -d \")
+export API_TOKEN
+
```

Comment:
Add "if file exists" here, perhaps.

Reply:
@Kris-LIBIS I added the "if" in this commit: 4f8ea658ce

Please take a look! Thanks!

Reply:
Looks great.